engine/src/main/java/com/badlogic/gdx/graphics/g2d/Animation.java

```diff
@@ -0,0 +1,216 @@
+/*Taken from the LibGDX source code from https://github.com/libgdx/libgdx on 03/11/2018*/
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
```

Comment:
What file?

Reply:
In terms of the code file: https://github.com/libgdx/libgdx/blob/57c466a8d294b74f7ca2377048795415714a9284/gdx/src/com/badlogic/gdx/graphics/g2d/Animation.java.
In terms of the AUTHORS file: https://github.com/libgdx/libgdx/blob/c31fe0ad74a35f4b1e898885615f5b91ee382953/AUTHORS.

I just copied the entire file directly from its source, so the only modification I made was to add the note at the top about where it came from.

Should I include the URLs to those files in the comment, or should I just paste the two names mentioned in the AUTHORS file directly into the code?

Reply:
Link to the file as part of your comment atop the file would be great